src/services/sections/utils/diffToMongo.js

```diff
@@ -0,0 +1,63 @@
+function isObject(o) {
+	return typeof o === 'object' && !Array.isArray(o) && o !== null;
+}
+
+function diffToMongo(diff, path = '') {
```

Comment:
kann doch auch ruhig in dem ordner utils bleiben? Ist zwar section bezogen... lässt sich drüber streiten